### PR TITLE
Add replication cli, and finished the dorado replication debuging

### PR DESCRIPTION
**Pull request by xxwjj**
**What this PR does / why we need it**:
1. add replication to cli
2. Finished the dorado replcation debuging.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/model/replication.go [hunk 1]
```diff
@@ -22,18 +22,26 @@ import "github.com/opensds/opensds/pkg/dock/proto"
...
+	ReplicationFailingOver    = "failingover"
+	ReplicationAvailable      = "available"
+	ReplicationError          = "error"
+	ReplicationErrorDeleting  = "error_deleting"
+	ReplicationErrorEnabling  = "error_enabling"
+	ReplicationErrorDisabling = "error_disabling"
+	ReplicationErrorFailover  = "error_failover"
+	ReplicationEnabled        = "enabled"
+	ReplicationDisabled       = "disabled"
+	ReplicationFailover       = "failover"
```
Comment (xing-yang) [inline pkg/model/replication.go]: failedover

**Comment by xing-yang** [comment]
Merge this now so Roland can use Replication CLI for testing.

**Comment by xxwjj** [comment]
@xing-yang  The replication unit tests are not available right now, I will add it later.

File: pkg/model/replication.go (hunk 1 above)
Comment (xxwjj) [reply pkg/model/replication.go]: ok, got it.

**Pull request merged**